rules-reviewed/eap6/uncategorized/generic-catchall.windup.xml

```diff
@@ -30,10 +30,10 @@
                         </not>
                     </when>
                     <perform>
-                        <classification title="Doug Lea Concurrency util" severity="mandatory">
+                        <classification title="Doug Lea Concurrency util" severity="potential">
```

Comment:
Optional and not potential? Potential is more than optional? And does "potential" imply that it's a mandatory change? I'm a bit confused.